### PR TITLE
new guilty mood

### PR DESCRIPTION
**Pull request by my-otter-self**
adds a new mood, the "i feel guilty".

### CODE REVIEW

File: Monika After Story/game/script-moods.rpy [hunk 1]
```diff
@@ -939,3 +939,25 @@ label mas_mood_grateful_gratefulforyou:
...
+            category=[store.mas_moods.TYPE_BAD],
+            unlocked=True
+        ),
+        code="MOO"
+    )
+
+label mas_mood_guilty:
+    m 1wkd "[player]! I'm sure you don't need to feel that way."
+    m 2fkb "The way for people to come to understand each other is forgiveness."
+    m 1dkc "We all make mistakes. "
```
Comment (dreamscached) [inline Monika After Story/game/script-moods.rpy:958]:
```suggestion
    m 1dkc "We all make mistakes.{w=0.3}{nw} "
```

File: Monika After Story/game/script-moods.rpy [hunk 2]
```diff
@@ -939,3 +939,25 @@ label mas_mood_grateful_gratefulforyou:
...
+    extend 1fka "And I'm sure you can be forgiven of whatever has happened."
+    m 7hub "After all, you're a great person. "
```
Comment (dreamscached) [inline Monika After Story/game/script-moods.rpy:960]:
```suggestion
    m 7hub "After all, you're a great person.{w=0.3}{nw} "
```

**Review by dreamscached — COMMENTED** [review]
Add missing waits

**Review by multimokia — DISMISSED** [review]

**Review by jmwall24 — APPROVED** [review]

**Review by jmwall24 — APPROVED** [review]

**Review by Booplicate — APPROVED** [review]
I approve that @jmwall24 is a qob

**Review by multimokia — APPROVED** [review]
Go to bed

**Pull request merged**